app/views/concerns/paginated_json_api_result_set_view.rb

```diff
@@ -59,4 +59,8 @@ def pagination_total_pages
       end
     end
   end
+
+  def pagination_page_item_count
```

Comment:
If I don't override this and implement either what is currently in the concern or your suggestion, the json responses will throw an exception:
`undefined method 'docs' for #<JsonApiClient::ResultSet:0x005626b49f7b40>`
Either way we'll need to override the method somewhere between the search results and the JSON API repsonses, they're structured differently.
Personally I think the default pagniated_view concern method should work for search results, as that is the portal's primary functionality, then we can just leave this override here.
The other place the pagination is used is in the galleries, which actually don't ever check for the paginated result number, however if they did, the same override as here `paginated_set.count` would work for them, but `paginated_set.docs.count` wouldn't. That's an argument for using `paginated_set.count` in the pagniated_view concern, and overriding the portal/index.rb view's method instead.
What do you think?